src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1146,14 +1192,29 @@ Node Visitor::visitTypesafe(Parser::BlankNodePropertyListContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-Node Visitor::visitTypesafe(Parser::CollectionContext* ctx) {
+void Visitor::visit(Parser::TriplesNodePathContext* ctx) {
+  // TODO: should be moved into the children. But visitAlternative does not
+  //  support returning void. Using the return type of the other branch is
+  //  not sensible in this case.
```

Comment:
I think we can make `visitAlternative` support `void` without too many complications and then get rid of this TODO.